swlor2_mdl/pmh0_belt153.mdl

```diff
@@ -0,0 +1,10718 @@
+# Exported from NeverBlender 2.8.33 - Sunday, 2022-05-08
+filedependancy ARMORPACK.blend
```

Comment:
One here too

Reply:
This model actually is crashing my game when trying to load it. On the rare occasion I CAN load it, it's got gunky stretched polygons. Gonna have to leave it out of the PR until I can figure out how to fix it.